src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const ifMarried = (sex === 'm'
+    ? `My wife name is ${partnerName}`
+    : `My husband's name ${partnerName}`);
```

Comment:
you can shorten it if you do it just for the word inside a string `wife/husband`